flake.nix

```diff
@@ -129,8 +129,8 @@
               
               use_influx
               
-              WT_METRICS_DIR="$(pwd)/telegraf/metrics"
-              export WT_METRICS_DIR
+              # shellcheck disable=SC1091
+              source ./scripts/telegraf.sh
```

Comment:
WT_METRICS_DIR is set the same way in telegraf.sh and that file is need to call import_hc_metrics later down.